backend/src/main/java/com/redhat/cloud/notifications/processors/email/OldDailyEmailCronJob.java

```diff
@@ -0,0 +1,158 @@
+package com.redhat.cloud.notifications.processors.email;
+
+import com.redhat.cloud.notifications.db.EmailAggregationResources;
+import com.redhat.cloud.notifications.db.EndpointEmailSubscriptionResources;
+import com.redhat.cloud.notifications.ingress.Action;
+import com.redhat.cloud.notifications.models.EmailAggregation;
+import com.redhat.cloud.notifications.models.EmailAggregationKey;
+import com.redhat.cloud.notifications.models.Endpoint;
+import com.redhat.cloud.notifications.models.NotificationHistory;
+import com.redhat.cloud.notifications.processors.EndpointTypeProcessor;
+import com.redhat.cloud.notifications.templates.EmailTemplate;
+import com.redhat.cloud.notifications.templates.EmailTemplateFactory;
+import com.redhat.cloud.notifications.transformers.BaseTransformer;
+import io.quarkus.qute.TemplateInstance;
+import io.quarkus.scheduler.Scheduled;
+import io.quarkus.scheduler.ScheduledExecution;
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+import io.smallrye.mutiny.tuples.Tuple2;
+import org.eclipse.microprofile.config.ConfigProvider;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.logging.Logger;
+
+
+import static com.redhat.cloud.notifications.models.EmailSubscriptionType.DAILY;
+import static com.redhat.cloud.notifications.models.EmailSubscriptionType.INSTANT;
+import static com.redhat.cloud.notifications.models.EmailSubscriptionType.values;
+import static java.time.ZoneOffset.UTC;
+
+@ApplicationScoped
+public class OldDailyEmailCronJob implements EndpointTypeProcessor {
+
+    private final Logger log = Logger.getLogger(this.getClass().getName());
+
+    @Inject
+    EndpointEmailSubscriptionResources subscriptionResources;
+
+    @Inject
+    RecipientResolver recipientResolver;
+
+    @Inject
+    BaseTransformer baseTransformer;
+
+    @Inject
+    EmailTemplateFactory emailTemplateFactory;
+
+    @Inject
+    EmailAggregationResources emailAggregationResources;
+
+    @Inject
+    EmailSender emailSender;
+
+    @Inject
+    EmailSubscriptionTypeProcessor emailSubscriptionTypeProcessor;
+
+    @Override
+    public Multi<NotificationHistory> process(Action action, List<Endpoint> endpoints) {
```

Comment:
This methods processes `instant` emails also, it should remain in `EmailSubscriptionTypeProcessor`.